### PR TITLE
Remove git when deploying to dev

### PR DESCRIPTION
**Pull request by loicgasser**
same as in https://github.com/geoadmin/mf-geoadmin3/pull/3174

we save 800Mb transfer when deploying with this approach.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ BRANCH_TO_DELETE :=
 CURRENT_DIRECTORY := $(shell pwd)
 DEPLOYCONFIG ?=
 DEPLOY_TARGET ?=
-GIT_BRANCH ?= $(shell git rev-parse --symbolic-full-name --abbrev-ref HEAD)
+GIT_BRANCH := $(shell if [ -f '.venv/deployed-git-branch' ]; then cat .venv/deployed-git-branch 2> /dev/null; else git rev-parse --symbolic-full-name --abbrev-ref HEAD; fi)
```
Comment (gjn) [inline Makefile]: This will remove the abbility to overwrite GIT_BRANCH on the cmdline. The README.md should reflect this (there's an example there).

Comment (loicgasser) [inline Makefile]: yes you're right forgot about this one

**Comment by loicgasser** [comment]
updated and rebased

**Comment by gjn** [comment]
Just a thought: how about also doing this after `rm -rf .git*`

```
git init
git add .
git commit -m "Initial commit"
```

Therefore, we could see if snapshot changed after it was created.

**Pull request merged**